ext/hal/nordic/Kconfig

```diff
@@ -19,6 +19,9 @@ config NRFX_ADC
 config NRFX_CLOCK
 	bool
 
+config NRFX_DPPIC
```

Comment:
While renaming the DT node to `dppic` is ok, because it refers to the DPPIC peripheral, this option should stay with the name `NRFX_DPPI`, since it enables nrfx driver `nrfx_dppi` (without the `c`).